publiq-ui/pub-radio-group.stories.mdx

```diff
@@ -0,0 +1,46 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import PubRadioGroup from './pub-radio-group.vue';
+
+<Meta title="Components/pub-radio-group" component={PubRadioGroup} />
+
+export const Template = (args, { argTypes }) => {
+  return {
+    props: Object.keys(argTypes),
+    components: { PubRadioGroup },
+    data() {
+      return {
+        selected: '',
+      };
+    },
+    template:
+      '<div><pub-radio-group v-bind="$props" v-model="selected" /><br/><span>Selected city: {{selected}}</span></div>',
+  };
+};
+
+export const commonArgs = {
+  name: 'city',
+  groupLabel: 'Select a city:',
+  labels: ['Rome', 'Paris', 'Prague'],
+  values: ['rome', 'paris', 'prague'],
+};
+
+## Default
+
+<Canvas>
+  <Story name="Default" args={commonArgs}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+```javascript
+<pub-radio-group
+  v-model="selected"
+  name="city"
+  groupLabel="Select a city:"
+  :labels="['Rome', 'Paris', 'Prague']",
+  :values="['rome', 'paris', 'prague']",
```

Comment:
The example in Storybook is broken since the latest changes to use a collection instead of two arrays.